test/nn/norm/test_agg_norm.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import torch
+
+from torch_geometric.nn import AggSubtraction
+
+
+def test_agg_norm():
```

Comment:
Why don‘t we call it MeanSubtractionNorm?

Reply:
just because I added the flexibility